pkg/kubectl/cmd/create_clusterrolebinding_test.go

```diff
@@ -0,0 +1,308 @@
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group1",
+					},
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group2",
+					},
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group3",
+					},
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: "rbac.authorization.k8s.io",
+					Kind:     "ClusterRole",
+					Name:     "clusterrole-test",
+				},
+			},
+		},
+		"test-multi-user-multi-group": {
+			clusterrole: "clusterrole-test",
+			users: []string{
+				"user1",
+				"user2",
+				"user3",
+			},
+			groups: []string{
+				"group1",
+				"group2",
+				"group3",
+			},
+			expectedClusterRoleBinding: &rbac.ClusterRoleBinding{
+				ObjectMeta: v1.ObjectMeta{
+					Name: clusterRoleName,
+				},
+				Subjects: []rbac.Subject{
+					{
+						Kind:     "User",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "user1",
+					},
+					{
+						Kind:     "User",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "user2",
+					},
+					{
+						Kind:     "User",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "user3",
+					},
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group1",
+					},
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group2",
+					},
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group3",
+					},
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: "rbac.authorization.k8s.io",
+					Kind:     "ClusterRole",
+					Name:     "clusterrole-test",
+				},
+			},
+		},
+		"test-dup-group": {
+			clusterrole: "clusterrole-test",
+			groups: []string{
+				"group1",
+				"group1",
+				"group1",
+			},
+			expectedClusterRoleBinding: &rbac.ClusterRoleBinding{
+				ObjectMeta: v1.ObjectMeta{
+					Name: clusterRoleName,
+				},
+				Subjects: []rbac.Subject{
+					{
+						Kind:     "Group",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "group1",
+					},
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: "rbac.authorization.k8s.io",
+					Kind:     "ClusterRole",
+					Name:     "clusterrole-test",
+				},
+			},
+		},
+		"test-dup-user": {
+			clusterrole: "clusterrole-test",
+			users: []string{
+				"user1",
+				"user1",
+				"user1",
+			},
+			expectedClusterRoleBinding: &rbac.ClusterRoleBinding{
+				ObjectMeta: v1.ObjectMeta{
+					Name: clusterRoleName,
+				},
+				Subjects: []rbac.Subject{
+					{
+						Kind:     "User",
+						APIGroup: "rbac.authorization.k8s.io",
+						Name:     "user1",
+					},
+				},
+				RoleRef: rbac.RoleRef{
+					APIGroup: "rbac.authorization.k8s.io",
+					Kind:     "ClusterRole",
+					Name:     "clusterrole-test",
+				},
+			},
+		},
+	}
+
+	for name, test := range tests {
+		buf := bytes.NewBuffer([]byte{})
```

Comment:
It would be nice to use 
```go
t.Run(test.name, func(t *testing.T) {
   // test body goes here
})
``` 
here, you won't have to deal with test names anywhere in your error reports.